components/dropdown/src/styles/bibStyles.scss

```diff
@@ -11,23 +11,39 @@
 
 /* stylelint-disable no-descending-specificity, selector-max-attribute */
 
+:host {
+  position: absolute;
+  z-index: 1;
```

Comment:
Maybe we can move these two rules into this select `:host([data-show])` since they don't do anything until then. What do you think?

Reply:
Changing `position` style would make the browser to re-calculate of the layer's position, which affects the performance. I'd not recommend.